src/test/java/bio/overture/ego/service/DefaultProviderServiceTest.java

```diff
@@ -0,0 +1,50 @@
+package bio.overture.ego.service;
+
+import static org.junit.Assert.*;
+
+import bio.overture.ego.model.enums.ProviderType;
+import bio.overture.ego.repository.DefaultProviderRepository;
+import bio.overture.ego.utils.EntityGenerator;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@Slf4j
+@SpringBootTest
+@RunWith(SpringRunner.class)
+@ActiveProfiles("test")
+public class DefaultProviderServiceTest {
+
+  @Autowired DefaultProviderService defaultProviderService;
+  @Autowired DefaultProviderRepository defaultProviderRepository;
+  @Autowired private EntityGenerator entityGenerator;
+
+  @Value("${spring.flyway.placeholders.default_provider}")
+  private String configuredDefaultProvider;
+
+  @Test
+  public void defaultProviderConfigured_Success() {
+    assertNotNull(configuredDefaultProvider);
+    defaultProviderService.findById(configuredDefaultProvider);
+    val defaultProvider = defaultProviderService.getById(configuredDefaultProvider);
+    assertNotNull(defaultProvider);
+    assertEquals(defaultProvider.getId(), configuredDefaultProvider);
+  }
+
+  @Test
+  public void defaultProviderConfigured_Failure() {
```

Comment:
just an fyi, to truly test the tripwire, we need to try and run spring, and midway update its context, so it can trigger the context refresh event and see the trip wire work

Reply:
you can try something like this
https://stackoverflow.com/a/55242370

Reply:
@anncatton timebox this to a day. If not achievable, we will just validate in Dev/Qa